client/components/data/query-theme/README.md

```diff
@@ -0,0 +1,58 @@
+Query Themes
+============
+
+Query Themes is a React component used in managing the fetching of themes queries.
+
+## Usage
+
+Render the component, passing `siteId` and `themeId`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```
+import React from 'react';
+import { connect } from 'react-redux';
+import QueryTheme from 'components/data/query-theme';
+import Theme from 'components/theme';
+import { getThemeById } from 'state/themes/selectors';
+
+function MyTheme( { themes } ) {
+	return (
+		<div>
+			<QueryTheme
+				siteId={ 3584907 }
+				themeId={ 'twentysixteen' } />
+				<Theme
+					key={ theme.id }
+					theme={ theme } />
+			} }
+		</div>
+	);
+}
+
+export default connect(
+	( state, { themeId } ) => ( {
+		theme: getThemeById( state, themeId )
```

Comment:
should be

```js
theme: getTheme( state, 3584907, 'twentysixteen' )
```

since both `siteId` and `themeId` are explicitly set in the example anyway. This means we can also drop the `{ themeId }` arg.